servo-pca9685/src/main.rs

```diff
@@ -0,0 +1,32 @@
+#![feature(alloc_system)]
+extern crate alloc_system;
+
+extern crate servo_pca9685;
+extern crate tessel;
+
+use servo_pca9685::ServoArray;
+use tessel::Tessel;
+use std::thread::sleep;
+use std::time::Duration;
+
+fn main() {
+    // Acquire port A.
+    let (port_a, _) = Tessel::ports().unwrap();
+
+    // Create the accelerometer object and connect to the sensor.
+    let mut servos = ServoArray::new(port_a, false, false);
```

Comment:
Without looking at the source, I would have no idea what `false, false` meant :\